README.md

```diff
@@ -79,6 +80,33 @@ If the source file does not follow the pattern `create[custom element]Element`,
 dojo build webpack --element=src/path/to/element.ts --elementPrefix=the-special
 ```
 
+### Feature optimization
+
+This command supports the ability to optimize code based on statically asserted features.  The tool can search the source code for modules that attempt to detect features using a [`@dojo/has`](https://github.com/dojo/has) type of API.  By supplying a feature set (or sets) on the command line, the build will be optimized to optimize code branches, making the code smaller and slightly more efficient.  This allows targeting of particular platforms.  The tool will not optimize code, when specifying multiple feature sets, they do not align.  It will instead continue to leave that feature to be detected at run-time.
```

Comment:
Move this to a separate paragraph and change "The tool will not optimize code, when specifying multiple feature sets, they do not align.  It will instead continue to leave that feature to be detected at run-time." to "When specifying multiple feature sets, if they do not align, the tool will not optimize the source code for these feature sets and will instead continue to leave that feature to be detected at run-time."